src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RequestJudgementByAdmissionForSpecCuiCallbackHandler.java

```diff
@@ -106,7 +106,7 @@ private CallbackResponse updateBusinessProcessToReady(CallbackParams callbackPar
             data.getCcjPaymentDetails();
 
         if (featureToggleService.isJudgmentOnlineLive()
-            && data.isLRvLipOneVOne()
+            && JudgmentsOnlineHelper.isNonDivergentForDJ(data)
```

Comment:
data.isOneVOne()
// this condition is not nonDivergent because out of scope 1v2 and 2v1 cannot do Judgment of admission?

Reply:
1v2 is possible here
immediately journey only.

Reply:
now change it to only 1v1